src/Utils/CamelCaseHelper.php

```diff
@@ -6,7 +6,15 @@ class CamelCaseHelper
 {
     public static function keysToCamelCase($input)
     {
-        $obj = new \stdClass();
+        if ($input instanceof \DateTime) {
+            return $input;
+        }
```

Comment:
We don't want to remap DateTime object - it caused the actual values to change as well